datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -3,23 +3,109 @@
 
 BEGIN;
```

Comment:
Please convert **all** your TABs to spaces in this file - it is very random now what has been used...